Locale/frFR/config.lua

```diff
@@ -2,180 +2,180 @@ QuestieLocale.locale['frFR'] = {
 	-- Trad by Pgmbru (V.5)
 	-- Configuration Windows
 	['OPTIONS_TAB'] = "Options",
-	['QUESTIE_HEADER'] = "Options Générales",
+	['QUESTIE_HEADER'] = "Options générales",
 	['ENABLE_QUESTIE'] = "Activer les icônes",
-	['ENABLE_QUESTIE_DESC'] = "Activer ou désactiver les icônes de Questie",
-	['ENABLE_ICON'] = "Activer le bouton de la mini-carte",
-	['ENABLE_ICON_DESC'] = "Activez ou désactivez le bouton de mini-carte de quête. Vous pouvez toujours accéder au menu des options avec Questie",
-	['ENABLE_INSTANT'] = "Activer le texte de recherche instantanée",
-	['ENABLE_INSTANT_DESC'] = "Active / Désactive l'option Instant Quest Text. Il ne s'agit que d'un raccourci pour l'option WoW dans Interface",
-	['LEVEL_HEADER'] = "Options de niveau de quête",
-	['ENABLE_LOWLEVEL'] = "Afficher toutes les quêtes situées en dessous de la plage (quêtes de niveau inférieur)",
-	['ENABLE_LOWLEVEL_DESC'] = "Activer ou désactiver l'affichage de l'affichage des quêtes de bas niveau sur la carte.",
-	['LOWLEVEL_BELOW'] = "<Afficher le niveau sous le niveau>",
-	['LOWLEVEL_BELOW_DESC'] = "Combien de niveaux sous votre personnage à afficher. (Par défaut: %s",
-	['LOWLEVEL_ABOVE'] = "<Afficher le niveau supérieur>",
-	['LOWLEVEL_ABOVE_DESC'] = "Combien de niveaux au-dessus de votre personnage à afficher. (Par défaut: %s",
-	['CLUSTER'] = "Quantité d'icônes d'objectif (pas encore implémenté)",
-	['CLUSTER_DESC'] = "Combien d'icônes d'objectif doivent être regroupées.",
-	['ENABLE_OBJECTIVES'] = "Activer les icônes d'objectif",
-	['ENABLE_OBJECTIVES_DESC'] = "Si cela est activé, les icônes d'objectif de quête seront affichées sur la carte / la mini-carte",
-	['ENABLE_TURNINS'] = "Activer les icônes de quête complétées",
-	['ENABLE_TURNINS_DESC'] = "Lorsque cette option est activée, les lieux d'activation de la quête seront affichés sur la carte / la carte minimale",
-	['ENABLE_AVAILABLE'] = "Activer les icônes de quête disponibles",
-	['ENABLE_AVAILABLE_DESC'] = "Lorsque cette option est activée, les lieux d'activation de la quête seront affichés sur la carte / la mini-carte",
+	['ENABLE_QUESTIE_DESC'] = "Active les icônes de Questie",
+	['ENABLE_ICON'] = "Activer le bouton autour de la mini-carte",
+	['ENABLE_ICON_DESC'] = "Active le bouton Questie autour de la mini-carte. Le panneau de configuration reste accessible avec la commande /questie.",
+	['ENABLE_INSTANT'] = "Activer Texte instantané (quêtes)",
+	['ENABLE_INSTANT_DESC'] = "Active l'option Texte instantané (quêtes). Cela revient à cocher l'option du même nom dans Interface/Affichage.",
+	['LEVEL_HEADER'] = "Options du niveau des quêtes",
+	['ENABLE_LOWLEVEL'] = "Afficher les quêtes obsolètes",
+	['ENABLE_LOWLEVEL_DESC'] = "Active l'affichage des quêtes de bas niveaux sur la carte.",
+	['LOWLEVEL_BELOW'] = "<Niveaux inférieurs max>",
+	['LOWLEVEL_BELOW_DESC'] = "Définis l'écart maximal entre le niveau des quêtes et celui du joueur. (Par défaut : %s)",
```

Comment:
I am not a fan of a whitespace before a colon. Could you change this and the below occurrences?

```suggestion
	['LOWLEVEL_BELOW_DESC'] = "Définis l'écart maximal entre le niveau des quêtes et celui du joueur. (Par défaut: %s)",
```